prettier.config.js

```diff
@@ -1,55 +1,93 @@
-// Modified from https://github.com/matzkoh/prettier-plugin-packagejson to use the custom sort-package.json package
-function prettier_plugin_packagejson() {
-  function requireSafe(path) {
-    try {
-      return require(path)
-    } catch (error) {
-      // no operations
+
+const plugins = [ require('prettier-plugin-jsdoc') ];
+
+
+/*
+ *  Modified from https://github.com/matzkoh/prettier-plugin-packagejson 
+ *  to use the custom sort-package.json package
+ */
+
+{
+    
+    function requireSafe(path){
+        try { return require(path); }
+        catch () {}
+    }
+    
+    
+    //  Static
+    
+    const isPackageJSON = /(^|\\|\/)package\.json$/;
+
+    const
+        parser_babylon = 'prettier/parser-babylon' ,
+        parser_babel = 'prettier/parser-babel' ,
+        package_sort = 'sort-package-json' ;
```

Comment:
Break this into separate declaration statements. Eslint should have warned you about this. 
`;` will be removed once you run pnpm format.